apps/explorer/src/pages/search-result/SearchResult.tsx

```diff
@@ -34,7 +34,7 @@ const querySearchParams = async (input: string, network: Network | string) => {
     if (
         isValidTransactionDigest(
             input,
-            version?.major === 0 && version?.minor < 18 ? 'base64' : 'base58'
+            'base58'
```

Comment:
Could you remove the unused variable on line 32? Also, we should remove the parameter `serializationFmt` from the signature of `isValidTransactionDigest` on https://github.com/MystenLabs/sui/blob/735bd4d0bd867bf7df9ae35c1d4c114bb9f68787/sdk/typescript/src/types/common.ts#L69 since we now exclusively use base58 for transaction digest